docker-compose.yaml

```diff
@@ -12,10 +12,10 @@ services:
     volumes:
```

Comment:
This one is pretty general. Do we have this docker-compose here to run the freva-web standalone for dev purposes? Because for deployment, it installs all services which are mentioned here one by one, and as I understood i think we don't use this docker compose in freva deployment at all. procedure there is totally different. This one popped up out of curiosity

Reply:
Yes, it's only for setting up the needed services for a development environment on your local computer. Nothing more.